components/loading.tsx

```diff
@@ -2,7 +2,7 @@ import { Flex, H3, Panel, ProgressCircle } from '@bigcommerce/big-design';
 
 const Loading = () => (
     <Panel>
-        <H3>Loading...</H3>
```

Comment:
since this is a loading indicator we should probably leave the wording as `loading`